### PR TITLE
Progress update: KSLHacks/SignalR_Chatroom

### PR DESCRIPTION
**Pull request by KSLHacks**
Progress update so we can sync and get the newest version of dev branch to corporate EF.  Still working on Issue #41 for Basic Chatroom working. Opening Issue #49 (Implementing IRepository InMemory)

### CODE REVIEW

**Comment by KSLHacks** [comment]
Testing Use Case:
+Jabbr should load successfully (get past the R logo splash). 
+One room should be visible and the load room should be successful. (located in browser dev tools).  
+Upon clicking the room, open room should FAIL. Not implemented at this time.

**Comment by adamtuliper** [comment]
Can someone test this?

**Comment by JamesEarle** [comment]
Will do. There are also conflicts with dev, @KSLHacks I can look into that if you want.

**Comment by jcocchi** [comment]
:shipit:

File: src/JabbR-Core/Hubs/Chat.cs [hunk 1]
```diff
@@ -1,40 +1,68 @@
...
 
-
 namespace JabbR_Core.Hubs
 {
 
     public class Chat : Hub
     {
-        public List<string> Rooms;
+        // Mock model instances to pass into Hub methods
+        public List<string> ChatRooms;
```
Comment (adamtuliper) [inline src/JabbR-Core/Hubs/Chat.cs]: public should be properties w/ get;set;

File: src/JabbR-Core/ViewModels/RoomViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    {
+        public string Name { get; set; }
+        public int Count { get; set; }
+        public bool Private { get; set; }
+        public string Topic { get; set; }
+        public bool Closed { get; set; }
+        public string Welcome { get; set; }
+        public IEnumerable<UserViewModel> Users { get; set; }
+        public IEnumerable<string> Owners { get; set; }
+        //public IEnumerable<MessageViewModel> RecentMessages { get; set; }
```
Comment (adamtuliper) [inline src/JabbR-Core/ViewModels/RoomViewModel.cs]: Can we bring this view model over to complete this class?

File: src/JabbR-Core/Hubs/Chat.cs (hunk 1 above)
Comment (KSLHacks) [inline src/JabbR-Core/Hubs/Chat.cs]: Fixed.

File: src/JabbR-Core/ViewModels/RoomViewModel.cs (hunk 2 above)
Comment (KSLHacks) [inline src/JabbR-Core/ViewModels/RoomViewModel.cs]: MessageViewModel requires a few other models to implement. Figured we should keep it minimum until we find a need for basic chat to work. Or at least until we get the latest dev merge.

**Pull request merged**